Client/Frontend/AuthenticationManager/PasscodeEntryViewController.swift

```diff
@@ -99,6 +99,7 @@ extension PasscodeEntryViewController: PasscodeInputViewDelegate {
             authenticationInfo?.recordValidation()
             KeychainWrapper.sharedAppContainerKeychain.setAuthenticationInfo(authenticationInfo)
             delegate?.passcodeValidationDidSucceed()
+            passcodePane.codeInputView.resetCode()
```

Comment:
Since we also call this in the else flow just below, lets move both to be after the if-block so its executed regardless